process/block/preprocess/transactionsV2.go

```diff
@@ -227,7 +227,7 @@ func (txs *transactions) processTransaction(
 			"senderUsername", tx.GetSndUserName(),
 			"receiverAddr", tx.GetRcvAddr(),
 			"receiverUsername", tx.GetRcvUserName(),
-			"data", string(tx.Data),
+			"data", tx.Data,
```

Comment:
please revert the changes for the data field. it is directly convertible to string and the logger will display them as hex, which implies a further step when debugging

Reply:
applicable for the entire changeset of this commit